Gemfile

```diff
@@ -1,4 +1,13 @@
 source "https://rubygems.org"
 
+gem "bundler"
+gem "minitest"
+gem "pry"
+gem "rake"
+gem "rubocop"
+gem "simplecov"
+gem "simplecov-console"
+gem "webmock"
```

Comment:
Why do we put these here instead of the gemspec?

Reply:
Rubocop